public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -93,7 +93,7 @@ export function AppChrome({ children }: Props) {
           <LinkButton className={styles.skipLink} href="#pageContent">
             Skip to main content
           </LinkButton>
-          <header className={cx(styles.topNav)}>
+          <header className={cx(styles.topNav, isSingleTopNav && state.megaMenuOpen && styles.topNavMenuOpen)}>
```

Comment:
think we should only adjust this position when the menu is docked and open, not just open